### PR TITLE
fix(TCOMP-365): hide tJDBC component driver class name text input

### PR DESCRIPTION
**Pull request by wwang-talend**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TCOMP-365

**What is the new behavior?**
delete the driver class field

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

**Other information**:
https://jira.talendforge.org/browse/TCOMP-365

### CODE REVIEW

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/873/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/873/)

File: components-jdbc/src/main/java/org/talend/components/jdbc/datastore/JDBCDatastoreProperties.java [hunk 1]
```diff
@@ -97,7 +95,6 @@ public void setupProperties() {
         dbTypes.setValue(defaultDBType.id);
 
         jdbcUrl.setValue("\"" + defaultDBType.url + "\"");
```
Comment (vjaouen) [inline components-jdbc/src/main/java/org/talend/components/jdbc/datastore/JDBCDatastoreProperties.java]: Please, do not forget to remove the quotation mark --> " from the default db url.

Comment (wwang-talend) [inline components-jdbc/src/main/java/org/talend/components/jdbc/datastore/JDBCDatastoreProperties.java]: done, please check if it meet your requirement.

**Comment by vjaouen** [comment]
Integration tests are OK

**Pull request merged**